library/ssl_tls13_generic.c

```diff
@@ -965,6 +965,9 @@ static int ssl_tls13_postprocess_server_finished_message( mbedtls_ssl_context *s
         goto cleanup;
     }
 
+    /* randbytes is not used again */
+    mbedtls_platform_zeroize( ssl->handshake->randbytes,
```

Comment:
I would move this one in `mbedtls_ssl_tls13_generate_application_keys()` as well. It seems we have to keep those random numbers until this point for the purpose of exporting client/server application secrets and that way we can erase them just next to the last call using them.